config_defaults.yml

```diff
@@ -15,7 +15,7 @@
 Gemfile:
   supports_windows: false
   required:
-    ':development':
+    ':development, :unit_tests':
```

Comment:
This is another one of those "lore" items, but I believe `:development` was originally added because it is Gemfile tradition, and `:system_tests` and `:unit_tests` are what we actually wanted to use. Except I think we also only ever do `--without system_tests` and never reference development/unit_tests explicitly.